fireant/slicer/transformers/highcharts.py

```diff
@@ -95,12 +104,30 @@ def xaxis_options(self, dataframe, dim_ordinal, display_schema):
         }
 
     def yaxis_options(self, dataframe, dim_ordinal, display_schema):
-        axis_count = len(display_schema['metrics'].keys())
+        metrics_per_axis = defaultdict(int)
+        for _, metric in display_schema['metrics'].items():
+            metrics_per_axis[metric['axis']] += 1
 
-        # Axis for metrics are blue to facilitate differing those from the reference ones.
+        metric_idx = 0
```

Comment:
do the colors of the reference axes match the axes for the metrics?  it looks like this will just cause them to display the same colors in order